requirements.txt

```diff
@@ -1,23 +1,9 @@
 motor
 python-dotenv
-cdsapi
 fastapi
 pydantic
 pymongo
-requests
-pytz
-timezonefinder
-simplekml
-numericalunits
 jsonpickle
 gunicorn
 uvicorn
-urllib3<2
```

Comment:
Was that issue with `urllib` version resolved? Docker & the local python 3.11 works but would like to confirm still

Reply:
Yep, with docker we don't need to worry about this anymore